src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
```

Comment:
We're going to have to connect to the blob container without using a connection string. The issue here is that the deployment will be run on azure subscriptions that will not have access to the PCT central storage container. So we're going to need to access the blob files via the `request` module.